accounts/abi/bind/template.go

```diff
@@ -79,6 +79,13 @@ var tmplSource = map[Lang]string{
 	LangJava: tmplSourceJava,
 }
 
+// SetTmplSource supports this func in order to set special template file.
+func SetTmplSource(lang Lang, source string) {
+	if _, ok := tmplSource[lang]; ok {
```

Comment:
do we need this check?

Reply:
We need to check it, `abigen` can't translate the unsupported language.